src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,13 +28,21 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        
+        this.sequenceNumber = nextSequenceNumber;
+        Person.nextSequenceNumber += 1;
     }
 
     /**
      * Copy constructor.
      */
     public Person(ReadOnlyPerson source) {
         this(source.getName(), source.getPhone(), source.getEmail(), source.getAddress(), source.getTags());
+        //unsure if should implement sequenceNumber in ReadOnlyPerson interface.
```

Comment:
Comments should be about the code only. It should still make sense to any future developer reading the comment. If you want to tell something to the reviewer/tutor (but not future developers of the code), use a GitHub comment instead (like this comment I've added).